File: P1/P1S1E2/lib/CarreraMontania.rb
require('./ProductoCarrera.rb');

public
class CarreraMontania < ProductoCarrera

	# @ReturnType String
	public
	def toString()
		super + "\nTipo: Montania\nPorcentaje de retirada: " + self.porcentajeRetirada.to_s + "%"
	end

	public
	def initialize()
    super(20)
	end
end

File: P1/P1S1E1/src/P1S1E1/CarreraMontania.java
package P1S1E1;

public class CarreraMontania extends ProductoCarrera {

        public CarreraMontania(int nBicis, FactoriaAbstractaDeporte unaFact){
            super(nBicis, unaFact);
            super.setPorcentajeRetirada(20);
        }

        @Override
        public String toString(){
            return super.toString() + "\nTipo: Montania\nPorcentaje de retirada: " + super.getPorcentajeRetirada() + "%";
        }
}